ci/build_setup.sh

```diff
@@ -30,7 +30,7 @@ function setup_clang_toolchain() {
   echo "clang toolchain configured"
 }
 
-function setup_clang_libcxx_toolchain() {
+function setup_clang_toolchain() {
```

Comment:
Q: Is there any way to make this the default in the core .bazelrc options such that someone could disable it and go back to libstdcxx if they want? This would provide consistency across all builds in general, not just CI builds. I'm not sure if there is a good way to do this though?

/wait-any

Reply:
Good point, I will revise the script.